src/main/java/com/foundation/search/view/SearchView.java

```diff
@@ -11,81 +11,64 @@
  */
 
 package com.foundation.search.view;
-
-import java.awt.event.ActionListener;
-import java.awt.event.ActionEvent;
-import javax.swing.JButton;
-import javax.swing.JFrame;
-import javax.swing.JLabel;
-import javax.swing.JTextField;
-import javax.swing.JOptionPane;
+import javax.swing.*;
 
 /**
- * Class SearchView that will receive the data to search and will have communication with the controller
+ * This class is the main class to communicate with the controller.
+ * It contains the MainPanel and the settings of the Search window
  *
  * @author Emely LLanos
  * @version 1.0.
  */
+public class SearchView extends JFrame{
 
-public class SearchView extends JFrame implements ActionListener{
-
-    private JLabel nameLabel;
-    private JTextField nameInput;
-    private JLabel pathLabel;
-    private JTextField pathInput;
-    private JButton findButton;
+    private MainPanel mainPanel;
 
+    /**
+     * The constructor of ResultsPanel class where the components are set
+     * by invoking the setting() and init() method
+     */
     public SearchView() {
         super();
```

Comment:
Is there a reason why you are calling super() ?

Reply:
yes, to inherit the JFrame constructor